network-api/networkapi/wagtailpages/pagemodels/app_install.py

```diff
@@ -0,0 +1,110 @@
+from django.db import models
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+
+from .campaigns import CampaignPage
+
+
+class AppInstallPage(CampaignPage):
+
+    zen_nav = False
+
+    hero_heading = models.CharField(
+        max_length=80,
+        help_text="Hero story headline",
+    )
+    hero_subheading = models.CharField(
+        max_length=80,
+        blank=True,
+        help_text="Hero story subheadline",
+    )
+    hero_background = models.ForeignKey(
+        "wagtailimages.Image",
+        on_delete=models.PROTECT,
+        related_name="+",
+        help_text="Background image for the hero section",
+    )
+    hero_video = models.URLField(
+        null=True,
+        blank=True,
+        help_text="To find embed link: go to your YouTube video and click “Share,” then “Embed,” "
+        "and then copy and paste the provided URL only. EX: https://www.youtube.com/embed/3FIVXBawyQ",
+    )
+    button1_text = models.CharField(
+        max_length=50,
+        help_text="Text for Button 1",
+        null=True,
+        blank=True,
+    )
+    button1_download_link = models.URLField(
+        max_length=255,
+        help_text="Download link for Button 1",
+        null=True,
+        blank=True,
+    )
+    button1_image = models.ForeignKey(
+        "wagtailimages.Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Image for Button 1",
+    )
+    button2_text = models.CharField(
+        max_length=50,
+        help_text="Text for Button 2",
+        null=True,
+        blank=True,
+    )
+    button2_image = models.ForeignKey(
+        "wagtailimages.Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Image for Button 2",
+    )
+    button2_download_link = models.URLField(
+        max_length=255,
+        help_text="Download link for Button 2",
+        null=True,
+        blank=True,
+    )
```

Comment:
Seems to me like these buttons can be condensed down to a re-usable StreamField block. Do we have one? That approach would also help reduce duplicated code in `network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/app_install_hero.html` by allowing us to loop through the data.

Reply:
Thanks @robdivincenzo! This is an excellent idea, I have created the block [AppInstallDownloadButtonBlock](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/bb8fe7a0290ccf70105cbe4d384c1786b185c800/network-api/networkapi/wagtailpages/pagemodels/customblocks/app_install_download_button_block.py) and am now using it in the latest commit. 👍